src/main/java/io/zeebe/dmn/DmnJobHandler.java

```diff
@@ -46,9 +46,11 @@ public DmnJobHandler(DmnRepository repository, DmnEngine dmnEngine) {
   public void handle(JobClient client, ActivatedJob job) {
 
     final DmnDecision decision = findDecisionForTask(job);
+    System.out.println(decision);
     final Map<String, Object> variables = job.getVariablesAsMap();
-
+    System.out.println(variables);
```

Comment:
Maybe we can remove these lines of test code.

Reply:
@ikheraj17 could you please remove the sysouts?

Reply:
@falko Looking at the error log is not caused by this.

Reply:
@falko I will do so now